test/.eslintrc.json

```diff
@@ -8,6 +8,7 @@
 		"sinon": true
 	},
 	"rules": {
-		"arrow-body-style": "off"
+		"arrow-body-style": "off",
+		"no-unused-expressions": "off"
```

Comment:
Why do we need this?

Reply:
It's because of the chai, where we need like `to.be.null` etc